Allow Unix Socket to be configured for MySQL in maintenance scripts, and use by default.  Also replace split() with explode() in LDAP.php for PHP7.0 compatibility.

dancapper opened the pull request.

dancapper committed:
Allow use of sockets - homer_mysql_rotate.pl

Allow use of mysql socket to connect to database

dancapper committed:
Allow use of sockets

 - homer_mysql_remove_partitions.pl

dancapper committed:
Allow use of sockets

- rotation.ini

dancapper committed:
Fixed some syntax errors

dancapper committed:
Merge remote-tracking branch 'origin/master'

dancapper committed:
Merge remote-tracking branch 'origin/master'

dancapper committed:
Merge remote-tracking branch 'origin/master'

dancapper committed:
Merge remote-tracking branch 'origin/master'

dancapper committed:
Allows use of a Unix Socket to communicate with MySQL and uses it by
default

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/sipcapture/homer-api?pullRequest=127) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/sipcapture/homer-api?pullRequest=127) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: dancapper<br/>:x: dcapper<hr/>**dcapper** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

dancapper committed:
Change one instance of split() to explode() for PHP 7.0 Compatibility as
split() was deprecated in 5.3.0 and removed in 7.0.0
http://php.net/manual/en/function.split.php

dancapper commented:
Seems to be a problem with the signing of the CLA; i am going to close and resubmit this request
